src/engine_render.c

```diff
@@ -8372,6 +8409,10 @@ static void do_map_who_for_thing(struct Thing *thing)
         }
         break;
     case 5:
+        // Hide flags if very zoomed out, for more atmospheric overview
+        if (camera_zoom < 2000) {
```

Comment:
Could you make a define for that. I think it would end up as a configurable value, and that more things would like to use this.